### ISSUE TITLE
How to organize data sets?

### ISSUE THREAD

**Issue by Kiritooo** [Lmy0217/FashionAI#1]
hello, I'm new  of the CV, I want to use your code to understand how to use dataset for pose estimation, but I don't know  how to deal with data sets and name, I will not submit results to match only for learning, can you give a README thank you!

**Comment by Lmy0217** [comment]
@Kiritooo We add README and will gradually improve it. If you want to know the approach of deal with data sets, please read `datasets.py` carefully.

**Comment by wuhaotju** [comment]
兄弟，线上能到多少？

**Comment by shiontao** [comment]
I find that the Accuracy is very low, what about you?

**Comment by shiontao** [comment]
`Train Epoch: 16 [7680/9056 (85%)]	Loss: 1.734629
Train Epoch: 16 [8320/9056 (92%)]	Loss: 1.775722
Train Epoch: 16 [8960/9056 (99%)]	Loss: 1.782988
Test set: Average loss: 2.9240, Accuracy: 369/2264 (16%)
Train Epoch: 17 [0/9056 (0%)]	Loss: 1.722289
Train Epoch: 17 [640/9056 (7%)]	Loss: 1.780630
Train Epoch: 17 [1280/9056 (14%)]	Loss: 1.787481
Train Epoch: 17 [1920/9056 (21%)]	Loss: 1.722810
Train Epoch: 17 [2560/9056 (28%)]	Loss: 1.876558
`
Is that normal?

**Comment by augusyan** [comment]
@rmfckaka I think it may be caused by overfitting, try image enhancement algorithm to enlarge your datasets. And the train loss increased , I guess u may take a smaller lr to train the model.

**Comment by wuhaotju** [comment]
@rmfckaka 我没有尝试

**Comment by aigeek** [comment]
分数百分之几？

**Comment by wzjiang** [comment]
我的准确率也不高，没参加比赛，想试一下这个任务，有加群交流的吗（102611529）
